### PR TITLE
LPS-66228 Template Display class name link leads to 404

### PR DESCRIPTION
**Pull request by marcellustavares**

### CODE REVIEW

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/edit_template_display.jspf [hunk 1]
```diff
@@ -566,8 +566,8 @@ private String _getPaletteItemTitle(HttpServletRequest request, String label, Cl
 
 	String javadocURL = null;
 
-	if (className.startsWith("com.liferay.portal")) {
-		javadocURL = "http://docs.liferay.com/portal/7.0/javadocs/";
+	if (className.startsWith("com.liferay.portal.kernal")) {
```
Comment (juangon) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/edit_template_display.jspf]: Shouldn't be this "kernel"?

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/efdb00fbdb...6bda1f9464

**Comment by brianchandotcom** [comment]
@marcellustavares @travisrcory see typo fix in 6bda1f94644f9fb68b698961a6e053c0bb25f824

@juangon good catch!

**Comment by travisrcory** [comment]
@juangon thanks for catching that! :)

**Comment by marcellustavares** [comment]
Thanks @juangon!

**Pull request closed without merging**